tests/handshake.rs

```diff
@@ -0,0 +1,141 @@
+#![cfg(feature = "handshake")]
+use std::net::TcpListener;
+use std::thread::{sleep, spawn};
+use std::time::Duration;
+use tungstenite::error::{Error, ProtocolError, SubProtocolError};
+use tungstenite::handshake::client::generate_key;
+use tungstenite::handshake::server::{Request, Response};
+use tungstenite::{accept_hdr, connect};
```

Comment:
```suggestion
use std::{
    net::TcpListener,
    thread::{sleep, spawn},
    time::Duration,
};
use tungstenite::{
    accept_hdr, connect,
    error::{Error, ProtocolError, SubProtocolError},
    handshake::{
        client::generate_key,
        server::{Request, Response},
    },
};
```

Reply:
Since I have access to that repo, I've clicked merged for you.

Reply:
Thanks !
This should fix the CI !